imports/plugins/core/layout/client/templates/layout/header/header.js

```diff
@@ -16,38 +16,12 @@ Template.layoutHeader.events({
 });
 
 Template.layoutHeader.helpers({
-  TagNav() {
```

Comment:
I removed this helper because it was using an undefined `ReactionUI` component and did not seem to be used (should all be React now anyway right?)